assets/js/components/DottedPagination/Arrow.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 
-function Arrow({ children, onClick }) {
+function Arrow({ children, onClick, datatestID }) {
   return (
     <div
+      data-testid={datatestID}
```

Comment:
Added this to test properly the change of SAP system in the details view. 
Otherwise, the tests looked super dependant on number of icons, styles, etc

Reply:
You did good, totally plus one on this :+1: